app/controllers/users_controller.rb

```diff
@@ -1,3 +1,34 @@
 class UsersController < ApplicationController
-  def new; end
+  before_action :find_user, only: :show
+
+  def show; end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "signup.message.success"
+      redirect_to @user
+    else
+      flash[:danger] = t "signup.message.fail"
+      render 'new'
+    end
+  end
+
+  private
+
+  def find_user
+    @user = User.find params[:id]
+    return if @user
+    flash[:danger] = t "signup.message.login"
```

Comment:
cái này chưa check rubocop đúng kh, sau return thêm 1 line
```suggestion
    return if @user
    
    flash[:danger] = t "signup.message.login"
```

Reply:
oke anh nhe